games/rust/entrypoint.sh

```diff
@@ -3,9 +3,20 @@ cd /home/container
 
 # Make internal Docker IP address available to processes.
 export INTERNAL_IP=`ip route get 1 | awk '{print $(NF-2);exit}'`
+export SRCDS_APPID=258550
```

Comment:
The Rust docker image is specifically built for Rust only (it's useless for any other app due to the wrapper), so it may not make sense to allow modification of the SRCDS_APPID.

Reply:
@Software-Noob Is this how you want it?